optaplanner-docs/src/main/asciidoc/QuickStart/CloudBalancingSolverConfiguration/CloudBalancingSolverConfiguration-section.adoc

```diff
@@ -2,7 +2,7 @@
 = Solver Configuration
 :imagesdir: ../..
 
-Take a look at the solver configuration:
+The solver configuration file determines how the solving process works; it is considered a part of the code. The file is named `examples/sources/src/main/resources/org/optaplanner/examples/cloudbalancing/cloudBalancingSolverConfig.xml`.
```

Comment:
That path is incorrect if you're building from source. It's only correct if you're building from the zip.

Reply:
This requires upstream/downstream conditioning. Will remove for this PR and get back after modular file breakup.

(Modular file breakup is likely the next step. I have a lot more changes done from peer review. I think it's logical to get the same file structure in first, so that then I can feed in the peer review changes ile-by-fule rather than move them between upstream and downstream files manually)